vocab/properties.json

```diff
@@ -1041,7 +1117,9 @@
     },
     "description": "Coordinate point from which you get the best view of something.",
     "label": "Best view point",
+    "labelPlural": "Best view points",
```

Comment:
should be "viewpoint" not "view point"

probably the name should be "bestViewpoint"

Reply:
I can't change the actual property name. But it is only used in old versions.